pkg/mount/mount.go

```diff
@@ -2,12 +2,46 @@ package mount // import "github.com/docker/docker/pkg/mount"
 
 import (
 	"sort"
+	"strconv"
 	"strings"
-	"syscall"
 
 	"github.com/sirupsen/logrus"
 )
 
+// Error records an error from mount or unmount operation
+type Error struct {
```

Comment:
Is there any reason to make this public?

Reply:
I don't know, but for some reason `os.PathError` et al were made public.

I'll make it private